assets/js/components/ChecksCatalog/ChecksCatalogNew.jsx

```diff
@@ -1,66 +1,81 @@
 /* eslint-disable react/no-array-index-key */
-import React, { useEffect } from 'react';
+import React, { useState, useEffect } from 'react';
 import { useSelector, useDispatch } from 'react-redux';
 
 import { groupBy } from '@lib/lists';
 
 import { getCatalog } from '@state/selectors/catalog';
 import { updateCatalog } from '@state/actions/catalog';
+import { providersList } from '../ProviderLabel/ProviderLabel';
 import CatalogContainer from './CatalogContainer';
 import CheckItem from './CheckItem';
+import ProviderSelection from './ProviderSelection';
 
+const providers = ['default'].concat(providersList.slice());
 // eslint-disable-next-line import/prefer-default-export
 export function ChecksCatalogNew() {
   const dispatch = useDispatch();
-
+  const [selected, setSelected] = useState(providers[0]);
+  const provider = selected;
   const {
     data: catalogData,
     error: catalogError,
     loading,
-  } = useSelector(getCatalog());
+  } = useSelector(getCatalog(provider));
 
   useEffect(() => {
-    dispatch(updateCatalog());
-  }, [dispatch]);
+    setSelected(providers[0]);
+  }, [providers[0]]);
 
   useEffect(() => {
-    getCatalog();
-  }, []);
+    if (provider === 'default') {
+      dispatch(updateCatalog());
+    } else {
+      dispatch(updateCatalog({ provider }));
+    }
+  }, [dispatch, provider]);
 
   return (
-    <CatalogContainer
-      onRefresh={() => dispatch(updateCatalog())}
-      isCatalogEmpty={catalogData.length === 0}
-      catalogError={catalogError}
-      loading={loading}
-    >
-      <div>
-        {Object.entries(groupBy(catalogData, 'group')).map(
-          ([group, checks], idx) => (
-            <div
-              key={idx}
-              className="check-group bg-white shadow overflow-hidden sm:rounded-md mb-8"
-            >
-              <div className="bg-white px-4 py-5 border-b border-gray-200 sm:px-6">
-                <h3 className="text-lg leading-6 font-medium text-gray-900">
-                  {group}
-                </h3>
+    <div>
+      <ProviderSelection
```

Comment:
mmm, in not sure if this `ProviderSelection` shouldn't go inside the `CatalogContainer`, to don't show the filter if the API to wanda fails